containers/deploy/foreman-proxy/deployments.yml

```diff
@@ -9,12 +9,12 @@
     - restart
 
 - name: Create Foreman Proxy deployment
-  openshift_v1_deployment_config:
+  openshift_raw:
       state: "{{ deployment_state }}"
       force: false
       resource_definition:
```

Comment:
Does this need to change?

Reply:
Now that I'm looking for it. It looks like this was missed in quite a few places. Does this need to be changed to `definition` everywhere?

Reply:
Not required, this still works and is supported from what I can tell from the test run.

Reply:
Okay, then I think we should probably be consistent rather than using `definition` in some places and `resource_definition` in others. I'm fine either way, but I think it's strange to have both.